docs-v2/integrations/all/aws.mdx

```diff
@@ -22,14 +22,48 @@ API configuration: [`aws`](https://nango.dev/providers.yaml)
 -   [How to create a user pool](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pool-as-user-directory.html)
 -   [OAuth-related docs](https://docs.aws.amazon.com/cognito/latest/developerguide/authorization-endpoint.html)
 -   [List of OAuth scopes](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-define-resource-servers.html)
+-   [IAM API reference](https://docs.aws.amazon.com/IAM/latest/APIReference/welcome.html)
 
 <Tip>Need help getting started? Get help in the [community](https://nango.dev/slack).</Tip>
 
 ## Connection configuration in Nango
 
-AWS uses different domain extensions codes for different regions, i.e `us-east-2` for the US East (Ohio) region, `us-east-1` for the US East (N. Virginia) region, or `eu-west-1` for Europe (Ireland) region.It also uses different subdomains for different `user pools`.
+- AWS uses different domain extensions codes for different regions, i.e `us-east-2` for the US East (Ohio) region, `us-east-1` for the US East (N. Virginia) region, or `eu-west-1` for Europe (Ireland) region.It also uses different subdomains for different `user pools`.
 
 
 ## API gotchas
+- To create a new connection in Nango for AWS IAM, you'll need an Access Key ID and a Secret Access Key. Use these to set up your connection, with the Access Key ID as the username and the Secret Access Key as the password. For more details on generating these keys, visit [Create an access key](https://docs.aws.amazon.com/IAM/latest/UserGuide/access-key-self-managed.html#Using_CreateAccessKey).
+- With this you can now use your basic auth credentials to sign you requests as follows. For more on how to sign your requests, please vist [AWS signature version 4 for API requests](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_sigv.html)
+
+```js
+    if ('username' in connection.credentials && 'password' in connection.credentials && 'region' in connection.connection_config) {
```

Comment:
@bodinsamuel @TBonnin Not saying we should do this now but I wonder if it would make sense to internalize into the platform this kind of logic at some point.

Reply:
for sure, I'm all for it

Reply:
Ok cool, thanks, I'll sync with Bastien from a product perspective on this.

Reply:
100% Potential solutions: have a `AWS_SIGN_V4` dedicated auth_mode or a proxy subattributes to specify pre-build authorization logic
```
    proxy:
        headers:
            Authorization: AWS_SIGN_V4

```